style.css

```diff
@@ -0,0 +1,100 @@
+html {
+  /* Setting a base font size of 10px give us easier rem calculations
+     Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+   */
+  font-size: 10px;

```

Comment:
Please note that always try to remove the comments from your code, by this way your code will be look more professional and clean. :smile: